pages/project/read/content/index.js

```diff
@@ -106,10 +138,21 @@ module.exports = merge({}, baseContent, {
   },
   manageAccess: {
     title: 'Guest access',
-    content: 'Allow colleagues to view this project. Administrators will have access by default.',
+    content: 'Allow colleagues to view this application. Administrators will have access by default.',
```

Comment:
Is there a version of this where the word `application` is different for a granted licence?